### PR TITLE
fix(Navbar): cross-browser consistency of menu items width on hover

### PR DESCRIPTION
**Pull request by kark**
#### Summary

[Deployment link](https://mc-15822.mc-preview.europe-west1.gcp.escemo.com/adnan-nodejs-test-project/welcome)
This solution worked for me in:
- Chrome 121.0.6167.160, 
- FF 122.0.1, 
- Edge 121.0.2277.112, 
- Safari Version 17.3.1 (19617.2.4.11.12)

### CODE REVIEW

**Comment by CarlosCortizasCT** [comment]
I've been testing the deployment preview and I still see some inconsistencies.

**Safari** 
The selected menu item icon gets more padding on the right when hovering the nav bar:
Screenshot 2024-02-12 at 10 31 19

**Firefox**
Both the selected menu item icon and the hovered one get more padding on the right when hovering:
Screenshot 2024-02-12 at 10 32 00

**Chrome**
Similar to Firefox, but also makes some menu entries text to jump into two lines when hovering it.
[Welcome - adnan-nodejs-test-project - Merchant Center.webm](https://github.com/commercetools/merchant-center-application-kit/assets/97907068/bf66e3f8-0c86-447b-a5a7-dbc40396d35a)

**Comment by kark** [comment]
> I've been testing the deployment preview and I still see some inconsistencies.
> 
> **Safari** The selected menu item icon gets more padding on the right when hovering the nav bar: Screenshot 2024-02-12 at 10 31 19
> 
> **Firefox** Both the selected menu item icon and the hovered one get more padding on the right when hovering: Screenshot 2024-02-12 at 10 32 00
> 
> **Chrome** Similar to Firefox, but also makes some menu entries text to jump into two lines when hovering it. [Welcome - adnan-nodejs-test-project - Merchant Center.webm](https://github.com/commercetools/merchant-center-application-kit/assets/97907068/bf66e3f8-0c86-447b-a5a7-dbc40396d35a)

About the bug - I'm not really sure how to proceed if that's environment dependant
About entries jumping into two lines - that's the effect of the on hover scaling of the icon. It's been like this before (of course now it's more prominent due to unwanted padding, so before scaling up the name fits in one line)

**Comment by CarlosCortizasCT** [comment]
> About the bug - I'm not really sure how to proceed if that's environment dependant

I'm not sure I understand. Do you mean you don't see the unexpected behaviour when testing the deployment preview link you shared?

> About entries jumping into two lines - that's the effect of the on hover scaling of the icon. It's been like this before (of course now it's more prominent due to unwanted padding, so before scaling up the name fits in one line)

Yes, I get it. What I don't understand yet is why it just happen in Chrome. I don't see this behaviour in Safari and Firefox.

**Comment by kark** [comment]
> > About the bug - I'm not really sure how to proceed if that's environment dependant
> 
> I'm not sure I understand. Do you mean you don't see the unexpected behaviour when testing the deployment preview link you shared?
> 

Not really.
I now noticed there is still some adjustment needed in Safari, but now I'm not really sure to address all the cross-browser issues that you're encountering

| Chrome  |  FF | Safari  | Edge  |
|---|---|---|---|
|   |  |   |   |

File: packages/application-shell/src/components/navbar/main-navbar.styles.ts [hunk 1]
```diff
@@ -85,7 +86,10 @@ const leftNavigationOpenStyles = css`
   .body__menu-open ${MenuListItem} {
     height: auto;
     min-height: ${NAVBAR.itemSize};
-    width: 100%;
+    width: calc(
+      ${NAVBAR.sublistIndentationWhenExpanded} - 2 *
+        ${uiKitDesignTokens.spacing25}
+    );
```
Comment (kark) [inline packages/application-shell/src/components/navbar/main-navbar.styles.ts:92]: I think it's now ready for review (and it has been re-deployed)
The main change lies in using fixed values for the menu list item's width, replacing the previous reliance on 100% width and hover-applied padding (to address any cross-browser inconsistencies)

**Comment by FilPob** [comment]
Thanks! Looks good to me!

**Review by FilPob — APPROVED** [review]

**Review by chloe0592 — APPROVED** [review]
I've tested all the browsers and didn't caught any inconsistencies. ✨  Thanks Kacper!

**Review by CarlosCortizasCT — APPROVED** [review]
Also looking good from my side 💯 

Thanks a lot 👍

**Pull request merged**